dev/start.bash

```diff
@@ -0,0 +1,5 @@
+kind create cluster
+make dev-setup
+DISABLE_WEBHOOKS=true make install
+sleep 10
```

Comment:
This stinks and not quite complete since I should also run `make run` after install, but still ironing out the how of that.